src/plugins/index_pattern_editor/public/components/preview_panel/status_message/status_message.tsx

```diff
@@ -88,7 +88,7 @@ export const StatusMessage: React.FC<StatusMessageProps> = ({
         <span>
           <FormattedMessage
             id="indexPatternEditor.status.noSystemIndicesWithPromptLabel"
-            defaultMessage="No Elasticsearch indices match your pattern."
+            defaultMessage="No data streams, indices, or index aliases match your pattern."
```

Comment:
Should this be:

No data streams, indices, or index aliases match your index pattern.